secret-sharing/src/pss.rs

```diff
@@ -37,22 +38,22 @@ impl<F: FftField> PackedSharingParams<F> {
     #[allow(unused)]
     pub fn new(l: usize) -> Self {
         let n = l * 4;
-        let t = l - 1;
-        debug_assert_eq!(n, 2 * (t + l + 1));
+        let t = l;
```

Comment:
in the above section you commented the following `(t, l, n)` and `(1, 2, 8)` but the code here sets `t = l`, I guess the docs needs updating or the code is wrong.